files/origin-components/apiserver-template.yaml

```diff
@@ -59,6 +61,7 @@ objects:
               path: /healthz
               port: 8443
               scheme: HTTPS
+        nodeSelector: "${{NODE_SELECTOR}}"
```

Comment:
Shouldn't this just be "${NODE_SELECTOR}" since its within an OCP template?

Reply:
No.  See https://docs.openshift.org/latest/dev_guide/templates.html#writing-parameters.